workflow/Snakefile

```diff
@@ -15,6 +18,9 @@ include: "rules/utilities.smk"
 include: "rules/qc.smk"
 include: "rules/alignment_variantcalling.smk"
 include: "rules/analysis.smk"
+include: "rules/bcl_convert.smk"
+
+# containerized: "docker://chabbytmd1/ampseeker:1.0"
```

Comment:
```suggestion
```

Reply:
Assuming we don't need this docker line :) Unless you think its useful? Have never used docker before, open to suggestions.

Reply:
Yes we don't need this docker. This was a mini side quest of mine 😅. For reproducibility, we are good with Conda I think.